packages/sdk/src/client.ts

```diff
@@ -110,23 +111,31 @@ export function SoundClient({
   }
 
   // All the minting schedules for a given edition, including past and future
-  async function mintSchedules({ editionAddress }: { editionAddress: string }): Promise<MintSchedule[]> {
+  async function mintSchedules({
+    editionAddress,
+    fromBlockOrBlockHash,
+  }: {
+    editionAddress: string
+    fromBlockOrBlockHash?: BlockOrBlockHash
```

Comment:
I wonder if it would make more sense for this to be an optional param when initializing the client, since it's unlikely a user would ever want to query on a per-function basis. wdyt @vigneshka ?

Reply:
yeah that's another option, but also having the block for the specific edition is possible right?

Reply:
or it always has to go to the logs of the root contract?

Reply:
yea that makes sense